tests/main.py

```diff
@@ -56,6 +56,7 @@ def run_additional_checks(argv=None):
 - role-without-backticks: Search roles without backticks.
 - trailing-whitespace: Check for trailing whitespaces at end of lines.
 - unbalanced-inline-literals-delimiters: Search for unbalanced inline literals delimiters.
+- resource-file-referenced: Check for unreferenced resource files.
```

Comment:
No need to list it here as it's part of "all the checkers defined in checkers/* files" mentioned below.